examples/03_connectivity/plot_confounds_removal.py

```diff
@@ -0,0 +1,170 @@
+"""
+Carefully defining the confounds
+================================
+
+This example shows the importance of confounds removal for functional
+connectivity.
```

Comment:
It would be great if you add more details. Such as you commented below "I wanted to show tips about the voxel-to-voxel connectivity distribution."

Reference for Friston et al parameter model ?